packages/analytics-js-plugins/src/errorReporting/index.ts

```diff
@@ -73,6 +74,9 @@ const ErrorReporting = (): ExtensionPlugin => ({
         const { component, tolerateNonErrors, errorFramesToSkip, normalizedError } =
           getConfigForPayloadCreation(error, errorState?.severityReason.type as string);
 
+        if (!isAllowedToBeNotified(error)) {
```

Comment:
Why don't we perform this check after creating the payload? By then, the error message would also have been apparent.

Reply:
Done